core/Pos/Util/Util.hs

```diff
@@ -68,15 +68,26 @@ module Pos.Util.Util
        -- ** Buildable Fortnight
 
        , dumpSplices
+
+       -- * Filesystem utilities
+       , ls
+       , lstree
+       , withTempDir
+       , directory
+       , sleep
```

Comment:
True enough, is more of a "general purpose/shell" one. Can you think of a better name for this group of utilities?